### ISSUE TITLE
Yalign module not found

### ISSUE THREAD

**Issue by sanjanasri** [machinalis/yalign#5]
Hi,

```
      I tried installing yalign in python 3.4 using `sudo pip3 install yalign` , it shows installed correctly, but when i try to import i get no module found, i also checked in  `sys.path`, i see the yalign in dist-packages. I dunno what I am doing wrong. Please help
```

**Comment by jmansilla** [comment]
I'm almost sure that yalign is not completely usable on python3.

With python2.7 I found no issues.

**Comment by sanjanasri** [comment]
Thank you. It did get installed  on windows python 2.7 32 bit . i am able
to import modules. but i see no output. Please do guide me

On Thu, May 5, 2016 at 7:29 PM, Javier Mansilla notifications@github.com
wrote:

> I'm almost sure that yalign is not completely usable on python3.
> 
> With python2.7 I found no issues.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/machinalis/yalign/issues/5#issuecomment-217161942

## 

Thanks and regards,

Sanjanasri J.P

**Comment by jmansilla** [comment]
Did you see the documentation? https://yalign.readthedocs.io/en/latest/
Even better, the tutorial: https://yalign.readthedocs.io/en/latest/tutorial.html

**Comment by sanjanasri** [comment]
Hi,

```
    This is the screenshot. I get nothing on cmd screen
```

On Sun, May 8, 2016 at 5:11 AM, Javier Mansilla notifications@github.com
wrote:

> Did you see the documentation? https://yalign.readthedocs.io/en/latest/
> Even better, the tutorial:
> https://yalign.readthedocs.io/en/latest/tutorial.html
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/machinalis/yalign/issues/5#issuecomment-217677261

## 

Thanks and regards,

Sanjanasri J.P

**Comment by sanjanasri** [comment]
Am i missing out something.. pls guide

On Sun, May 8, 2016 at 6:02 PM, Sanjanashree Palanivel <
sanjanashree@gmail.com> wrote:

> Hi,
> 
> ```
>     This is the screenshot. I get nothing on cmd screen
> ```
> 
> On Sun, May 8, 2016 at 5:11 AM, Javier Mansilla notifications@github.com
> wrote:
> 
> > Did you see the documentation? https://yalign.readthedocs.io/en/latest/
> > Even better, the tutorial:
> > https://yalign.readthedocs.io/en/latest/tutorial.html
> > 
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly or view it on GitHub
> > https://github.com/machinalis/yalign/issues/5#issuecomment-217677261
> 
> ## 
> 
> Thanks and regards,
> 
> Sanjanasri J.P

## 

Thanks and regards,

Sanjanasri J.P

**Comment by rafacarrascosa** [comment]
Hi @sanjanasri ,

We cannot see the screenshot you attached. Instead of attaching another, please tell us in detail what are you doing so we can try to help you.
Bear in mind that we need to replicate the error you are seeing to be able to help you.
So please add detail, be informate and try to put yourself in our shoes while receiving your feedback. 

Regards,

Rafael

**Comment by sanjanasri** [comment]
I worked with yalign-train using the corpus given in tutorial.tar.gz, the
model was created successfully resulting in two files "aligner.pickle and
metadata.json...but, when I tried to execute the command "yalign-align
en-es http://en.wikipedia.org/wiki/Antiparticle
http://es.wikipedia.org/wiki/Antipart%C3%ADcula > 2.txt" I am getting
nothing written in the file. I mean just an 0KB file.

 I do not know what I am mssing

On Tue, May 10, 2016 at 11:25 PM, Rafael Carrascosa <
notifications@github.com> wrote:

> Hi @sanjanasri https://github.com/sanjanasri ,
> 
> We cannot see the screenshot you attached. Instead of attaching another,
> please tell us in detail what are you doing so we can try to help you.
> Bear in mind that we need to replicate the error you are seeing to be able
> to help you.
> So please add detail, be informate and try to put yourself in our shoes
> while receiving your feedback.
> 
> Regards,
> 
> Rafael
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> https://github.com/machinalis/yalign/issues/5#issuecomment-218238225

## 

Thanks and regards,

Sanjanasri J.P

**Comment by simontite-capita-ti** [comment]
Similarly for me!

`yalign-align -a en -b es -f tmx en-es http://en.wikipedia.org/wiki/Antiparticle http://es.wikipedia.org/wiki/Antipart%C3%ADcula`

results in output

`<?xml version="1.0" ?>`
`<!DOCTYPE tmx SYSTEM "tmx14.dtd">`
`<tmx version="1.4">`
`<header datatype="PlainText" srclang="en" segtype="sentence" creationtool="MTrans"/>`
`<body>`
`</body>`
`</tmx>`

and using `-f plaintext` or not using the `-f` flag at all results in absolutely nothing.

Do these comments belong in issue #6? It seems to be the same thing...

**Comment by sanjanasri** [comment]
yep. Now I get the same result. But I am supposed to get parallel sentence
rite. How do i get that

On Mon, May 16, 2016 at 9:45 PM, Simon Tite notifications@github.com
wrote:

> Similarly for me!
> 
> yalign-align -a en -b es -f tmx en-es
> http://en.wikipedia.org/wiki/Antiparticle
> http://es.wikipedia.org/wiki/Antipart%C3%ADcula
> 
> results in output
> 
> <?xml version="1.0" ?>
> <!DOCTYPE tmx SYSTEM "tmx14.dtd">
> <tmx version="1.4">
> <header datatype="PlainText" srclang="en" segtype="sentence"
> creationtool="MTrans"/>
> <body>
> </body>
> </tmx>
> 
> and using -f plaintext or not using the -f flag at all results in
> absolutely nothing.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> https://github.com/machinalis/yalign/issues/5#issuecomment-219469186

## 

Thanks and regards,

Sanjanasri J.P

**Comment by sanjanasri** [comment]
Hi,

```
 It is gentle reminder to let you know that I am totally unaware of the
```

output (tmx file) that was created from yalign. How do i extract data. I
couldn't see the sentences. pLease help

On Mon, May 16, 2016 at 10:08 PM, Sanjanashree Palanivel <
sanjanashree@gmail.com> wrote:

> yep. Now I get the same result. But I am supposed to get parallel sentence
> rite. How do i get that
> 
> On Mon, May 16, 2016 at 9:45 PM, Simon Tite notifications@github.com
> wrote:
> 
> > Similarly for me!
> > 
> > yalign-align -a en -b es -f tmx en-es
> > http://en.wikipedia.org/wiki/Antiparticle
> > http://es.wikipedia.org/wiki/Antipart%C3%ADcula
> > 
> > results in output
> > 
> > <?xml version="1.0" ?>
> > <!DOCTYPE tmx SYSTEM "tmx14.dtd">
> > <tmx version="1.4">
> > <header datatype="PlainText" srclang="en" segtype="sentence"
> > creationtool="MTrans"/>
> > <body>
> > </body>
> > </tmx>
> > 
> > and using -f plaintext or not using the -f flag at all results in
> > absolutely nothing.
> > 
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly or view it on GitHub
> > https://github.com/machinalis/yalign/issues/5#issuecomment-219469186
> 
> ## 
> 
> Thanks and regards,
> 
> Sanjanasri J.P

## 

Thanks and regards,

Sanjanasri J.P

**Comment by simontite-capita-ti** [comment]
@sanjanasri 
This issue has been fixed, for me at least, please see the answer to issue #6.

**Comment by imrrahul** [comment]
@sanjanasri @simontite-capita-ti I need ur help guys currently I am working on extraction of parallel senetences from comparable corpora using Yalign tool but I am facing problem plzz help..... give me ur mail id so that I can communicate with you regarding this.
